components/DetailsPageButton/index.js

```diff
@@ -0,0 +1,11 @@
+import Link from "next/link"
+import { StyledButton } from "./DetailsPageButton.styled"
+
+export default function DetailsPageButton({game}){
+    return(
+    <>
+   
+    <StyledButton> <Link href={`/games/${game.id}`}>More Details</Link></StyledButton>
+   
+    </>)
+}
```

Comment:
It looks like you aren't using an automatic formatter (`Prettier`)? If so, please install and use it to make code more readable in general. Contact me if it's not working :-)

```suggestion
export default function DetailsPageButton({ game }) {
  return (
    <>
      <StyledButton>
        <Link href={`/games/${game.id}`}>More Details</Link>
      </StyledButton>
    </>
  );
}
```